collect_app/src/main/java/org/odk/collect/android/map/OsmMapFragment.java

```diff
@@ -338,37 +371,133 @@ protected void addMapLayoutChangeListener(MapView map) {
         return new MapPoint(geoPoint.getLatitude(), geoPoint.getLongitude(), geoPoint.getAltitude());
     }
 
-    protected static @NonNull MapPoint fromGeoPoint(@NonNull GeoPoint geoPoint, double sd) {
-        return new MapPoint(geoPoint.getLatitude(), geoPoint.getLongitude(), geoPoint.getAltitude(), sd);
+    protected static @NonNull MapPoint fromMarker(@NonNull Marker marker) {
+        GeoPoint geoPoint = marker.getPosition();
+        double sd = 0;
+        try {
+            sd = Double.parseDouble(marker.getSubDescription());
+        } catch (NumberFormatException e) { /* ignore */ }
```

Comment:
I think logging at least Timber.w would be good here? What do you think? We usually don't ignore exceptions.

Reply:
Okay!  I also fixed a similar ignored exception in GoogleMapFragment.